src/components/Forms/SuggestionBox.js

```diff
@@ -0,0 +1,192 @@
+import React, { useState } from 'react';
+import {
+  FormControl,
+  Flex,
+  useTheme,
+  FormLabel,
+  Input,
+  Textarea,
+  Text,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+  NumberIncrementStepper,
+  NumberDecrementStepper,
+} from '@chakra-ui/core';
+import PrimaryButton from '../Buttons/PrimaryButton';
+
+const encode = data => {
+  return Object.keys(data)
+    .map(key => encodeURIComponent(key) + '=' + encodeURIComponent(data[key]))
+    .join('&');
+};
+
+//how to intigrate with netlifys for handling
+//https://www.netlify.com/blog/2017/07/20/how-to-integrate-netlifys-form-handling-in-a-react-app/#form-handling-with-static-site-generators
+
+export default function SuggestionBox() {
+  const [topic, setTopic] = useState(null);
+  const [description, setDescription] = useState(null);
+  const [benifits, setBenifits] = useState(null);
+  const [urgency, setUrgency] = useState(5);
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [validationMessage, setValidationMessage] = useState(null);
+  const [submitted, setSubmitted] = useState(false);
+
+  const theme = useTheme();
+
+  const handleSubmit = event => {
+    const toSubmit = {
+      topic,
+      description,
+      benifits,
+      urgency,
+      name,
+      email,
+    };
+
+    //Custom Validation
+    const valuesToValidate = Object.values(toSubmit);
+    //Validates all required fields (initiated with null) are filled
+    if (valuesToValidate.includes(null)) {
+      setValidationMessage('All fields with * are required.');
+      return;
+    }
+
+    //posts to Netlify intigration
+    fetch('/', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+      body: encode({ 'form-name': 'contact', toSubmit }),
+    })
+      .then(() => console.log('Success!'))
+      .catch(error => console.log(error));
+
+    event.preventDefault();
+    setSubmitted(true);
+  };
+
+  if (submitted) {
+    return (
+      <Text fontSize="2xl" textAlign="center">
+        Thank you for your suggestion!
+      </Text>
+    );
+  }
+
+  return (
+    <FormControl
```

Comment:
It appears that this `FormControl` element is outputting a div instead of a `<form>` element. It appears that we may just need to manually wrap the form elements with `<form netilfy>` to get those form submissions working properly!

![Screen Shot 2020-08-20 at 11 54 11 AM](https://user-images.githubusercontent.com/1380536/90807548-df486400-e2db-11ea-9419-781a7f5e5b87.png)

Reply:
Hey @SeHarlan - I'm not sure if this is just a challenge of testing the form on the deployment preview, but the form still doesn't get dropped into Netlify.

Do you mind trying one more thing for me? After looking at the documentation https://docs.netlify.com/forms/setup/#html-forms it appears that we may need a couple of additional attributes on the form. Primarily `name` and `method`.

Could you try adding the following:

`name="suggestion-box"`

and

`method="POST"`

Let's give this a shot, and if we still aren't getting form submissions through the deploy preview, we'll merge and see what we get :)

Reply:
It looks like Netlify is properly recognizing the form, the honeypot and some Netlify specific elements seem to be populating. I'm going to merge this and test the submissions on production!